agent/xds/listeners.go

```diff
@@ -32,6 +32,10 @@ import (
 	"github.com/hashicorp/consul/logging"
 )
 
+const (
+	TProxyOutboundPort = 15001
```

Comment:
Might want to file a TODO here about possibly making this configurable somewhere.